rpcs3/module_verifier.cpp

```diff
@@ -0,0 +1,96 @@
+#include "stdafx.h"
+#include "module_verifier.hpp"
+
+#ifdef _WIN32
+
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#include <Windows.h>
+
+#include <util/types.hpp>
+#include <util/logs.hpp>
+#include <Utilities/StrUtil.h>
+#include <Utilities/StrFmt.h>
+
+LOG_CHANNEL(sys_log, "SYS");
+
+[[noreturn]] void report_fatal_error(std::string_view text, bool is_html, bool include_help_text);
+
+void WIN32_module_verifier::run_module_verification()
+{
+	struct module_info_t
+	{
+		std::wstring_view name;
+		std::string_view package_name;
+		std::string_view dl_link;
+	};
+
+	const std::vector<module_info_t> special_module_infos = {
+		{ L"vulkan-1.dll", "Vulkan Runtime", "https://sdk.lunarg.com/sdk/download/latest/windows/vulkan-runtime.exe" },
+		{ L"msvcp140.dll", "Microsoft Visual C++ 2015-2019 Redistributable", "https://aka.ms/vs/16/release/VC_redist.x64.exe" },
```

Comment:
Should be 2015-2022 Redistributable with URL https://aka.ms/vs/17/release/VC_redist.x64.exe

Reply:
updated